gems/aws-sdk-core/lib/aws-sdk-core/sso_bearer_provider.rb

```diff
@@ -0,0 +1,180 @@
+# frozen_string_literal: true
+
+module AWS::SDK::Core
+  # A refreshing identity provider that provides HTTP Bearer Identities from
+  # SSOOIDC. The SDK does not manage refreshing of
+  # the token value, but this can be done by running `aws login` with the
+  # correct profile.
+  #
+  #     # You must first run aws sso login --profile your-sso-profile
+  #     provider = AWS::SDK::Core::SSOBearerProvider.new(
+  #       sso_region: "us-east-1",
+  #       sso_session: "your-sso-session"
```

Comment:
Single quotes since I noticed other code snippet examples was changed to follow this format